notebooks/ImvizExample.ipynb

```diff
@@ -351,7 +353,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "c = SkyCoord('00h24m07.33s -71d52m50.71s')\n",
+    "c = SkyCoord('10h36m49.211s -58d35m56.871s')\n",
```

Comment:
Would be nice to have this actually center on some feature on the image rather than just area of blank sky.